jupyter-docker/jupyter_localize_extension.py

```diff
@@ -6,34 +6,76 @@
 from tornado.web import HTTPError
 from notebook.base.handlers import IPythonHandler
 from notebook.utils import url_path_join
+from datauri import DataURI
```

Comment:
Using this [library](https://pypi.python.org/pypi/python-datauri/) to parse data URIs